release/RELEASE.md

```diff
@@ -125,31 +125,32 @@ patches we backport to the 1.0 release branch).
 
     1. `cd create-daml-app`
 
-       1. `daml build`
-
-       1. `daml codegen js .daml/dist/create-daml-app-0.1.0.dar -o ui/daml.js`
-
        1. `daml start`
 
     1. In a new terminal, from the `ui` folder:
 
-       1. `npm install`
+       1. `npm install --frozen-lockfile`
```

Comment:
I don’t think this makes sense. This is the first call to `npm install`, you are generating the lockfile here.

Reply:
You're totally right. I was blindly following the GSG but apparently the GSG is wrong in this regard.